src/client/client.cpp

```diff
@@ -1387,19 +1386,6 @@ void Client::sendPlayerPos()
 	f32 movement_speed = player->control.movement_speed;
 	f32 movement_dir = player->control.movement_direction;
 
-	if (
-			player->last_position        == player->getPosition() &&
-			player->last_speed           == player->getSpeed()    &&
-			player->last_pitch           == player->getPitch()    &&
-			player->last_yaw             == player->getYaw()      &&
-			player->last_keyPressed      == keyPressed            &&
-			player->last_camera_fov      == camera_fov            &&
-			player->last_camera_inverted == camera_inverted       &&
-			player->last_wanted_range    == wanted_range          &&
-			player->last_movement_speed  == movement_speed        &&
-			player->last_movement_dir    == movement_dir)
-		return;
-
 	player->last_position        = player->getPosition();
```

Comment:
Do we still need to keep last_position, etc, in player?

Reply:
Some of them yes, because Lua API functions use them (`last_position` is used by `LocalPlayer:get_last_position`, the others are used by serverside getters). Taking a closer look the others are indeed unnecessary now. I've removed the unnecessary ones.